Iterable-iOS-SDK/Iterable-iOS-SDK/IterableAPI.m

```diff
@@ -361,4 +368,21 @@ - (void)trackPurchase:(NSNumber *)total items:(NSArray<CommerceItem> *)items dat
     }
 }
 
+- (NSString*)pushServicePlatformToString:(PushServicePlatform)pushServicePlatform{
+    NSString *result = nil;
+
+    switch(pushServicePlatform) {
+        case APNS:
+            result = @"APNS";
+            break;
+        case APNS_SANDBOX:
+            result = @"APNS_SANDBOX";
+            break;
+        default:
+            [NSException raise:NSGenericException format:@"Unexpected PushServicePlatform."];
```

Comment:
Seems like we should never throw since that might crash the app? I forgot what that guy said about asserts but... either assert or return an error